### DISCUSSION TITLE
Question: ABAP Cloud deployment only with user&pw?

### DISCUSSION THREAD

**Discussion by ulr1chA** [SAP/open-ux-tools#1658] [Q&A]
Hi,

we want to deploy Fiori applications with our pipeline on an ABAP cloud environment.
All options, Fiori deploy & deploy (@sap-ux/deploy-tooling), which I found only offering username and password.
Is that right? No other possibilities?

Thank you and have a nice weekend.

**Comment by T1mey** [comment]
@longieirl any info about the support of this?

Found a influence idea, please vote for it...
https://influence.sap.com/sap/ino/#/idea/283281/?section=sectionExperts

**Comment by tmackowski** [comment]
Hi @ulr1chA ,

Did you find a solution for this? I'm trying to deploy to ABAP BTP from CI/CD pipeline and still no luck.
